applications/forms.py

```diff
@@ -216,7 +216,8 @@ class Meta:
             'origin': 'Where are you joining us from?',
             'description': 'Why are you excited about %s?' % settings.HACKATHON_NAME,
             'projects': 'What projects have you worked on?',
-            'resume': 'Upload your resume',
+            'resume': 'Upload your resume %s' % ('' if not settings.SUPPORTED_RESUME_EXTENSIONS
```

Comment:
`settings.SUPPORTED_RESUME_EXTENSIONS` will fail if if `SUPPORTED_RESUME_EXTENSIONS` is not set in the settings. Use `getattr(settings, 'SUPPORTED_RESUME_EXTENSIONS', None)` instead